indico/modules/events/abstracts/blueprint.py

```diff
@@ -95,6 +99,14 @@
 
 # Abstract-specific management
 _bp.add_url_rule('/manage/abstracts/<int:abstract_id>/', 'manage_abstract', RHManageAbstract, methods=('GET', 'POST'))
-_bp.add_url_rule('/manage/abstracts/<int:abstract_id>/reset',
-                 'reset_abstract_judgment', RHResetAbstractJudgment, methods=('POST',))
-_bp.add_url_rule('/manage/abstracts/<int:abstract_id>/abstract.pdf', 'manage_abstract_pdf_export', RHAbstractExportPDF)
+
+for prefix in ('/manage/abstracts', '/abstracts'):
```

Comment:
Can't we use the `add_prefixed_rules(...)` context like we do here https://github.com/indico/indico/blob/master/indico/modules/events/timetable/blueprint.py#L55?